.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: ruby
+cache: bundler
+bundler_args: "--jobs 7"
+
+rvm:
+  - 2.1.5
```

Comment:
You should probably use 2.1 and 2.2 though and let it float, similarly to the chef project, minor version pins usually wind up testing against old versions.